rich/pretty.py

```diff
@@ -731,7 +753,23 @@ def iter_attrs() -> Iterable[
                     append(child_node)
 
                 pop_visited(obj_id)
-
+        elif _is_namedtuple(obj):
```

Comment:
We are also going to need a check to see if the default `__repr__` was overriden. If it was, Rich should respect that. Have a look at `_is_dataclass_repr`, maybe we can do something similar for NamedTuple.